walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -57,7 +60,7 @@ public void setUp() {
   }
 
   @Test
-  public void optionsForNoAttendees() {
+  public void query_allDayAvailable_optionsForNoAttendees() {
```

Comment:
I am not sure I know what optionsForNoAttendees means if allDayAvailable?

Reply:
For this test, if there no attendees, the entire day is available for booking
I renamed the test methods such that the outcome (`allDayAvailable`) is at the end of the method name so hopefully, it is clearer. --> `query_noAttendees_allDayAvailable()`